pkgs/tools/text/rst2html5/default.nix

```diff
@@ -1,19 +1,22 @@
-{ stdenv, fetchurl, pythonPackages }:
+{ lib, python3Packages }:
 
-pythonPackages.buildPythonPackage rec {
+let
   pname = "rst2html5";
-  version = "1.9.4";
+  version = "1.10.6";
```

Comment:
Why are you always using let in? rec is 2 lines shorter and has less repetition. This makes only sense to use when you input another derivation, an icon or something similar.

Reply:
`rec` scrambles the scope of its variables, and I don't think it is bad to "factor" a variable like `pname`.